content/en/docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -124,69 +70,41 @@ The master is the machine where the control plane components run, including
 etcd (the cluster database) and the API server (which the kubectl CLI
 communicates with).
 
-To initialize the master, first choose the pod network plugin you want and check if it requires any parameters to be passed to kubeadm while initializing the cluster. Pick one of the machines you previously installed
-kubeadm on, and run:
+1. Choose a pod network add-on and verify if it requires any arguments to 
```

Comment:
What about changing the numbered list into:

"Before initializing your master, you should choose a pod network...."
and
"Please note that unless otherwise ..."